packages/slice-machine/src/legacy/lib/builders/common/EditModal/index.jsx

```diff
@@ -234,6 +234,8 @@ const EditModal = ({ close, data, fields, onSave, zoneType }) => {
                   />
                   <Close // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-assignment
                     onClick={close}
+                    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                    onMouseDown={close} // will execute the close function before Label input hijacks the focus
```

Comment:
I wonder if this is the best way to solve this, it looks like a hack. This is a common scenario on the web (clicking a button while another element has the focus). We rarely have to duplicate the on-click handler to fix this, and with this, we're executing the close function twice (it might be ok for this particular scenario, but it is still not ideal).

Reply:
I'm not 100% sure about what is exact root of the problem, it's most likely related with re-render happening when input is validated, and my idea was that after the re-render focus is brought back to input (if I disable autofocus, button works without any problem). 
I agree the solution looks like a hack, but couldn't find any other working (might try to spend some more time on it).

Reply:
I didn't find solution that would not require two handlers, but prevented firing close function twice

Reply:
@dani-mp I manage to get rid of those 2 event listeners by preventing the default field validation on blur (but not 100% sure if this is what we want)